### PR TITLE
45 adjust accepted format

### LINKED ISSUE #45: Adjust Accepted Document Format
**Acceptance Criteria**
- [ ] Frontend and Backend will accept the newly provided excel document and all documents with a similar format.

### PR DESCRIPTION
**Pull request by AaronPMichael**
**Addressed Issue(s)**

closes #45 

## Acceptance Criteria
- [ ] Backend now accepts the new excel format

### Special notes for reviewer
Backend no longer accepts the old excel format. Backend cannot process one of the dates in the example document ("9-Aug"). We will fix that later.

### CODE REVIEW

File: backend-processing.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export default class BackendProcessing {
     let con = await (this.create_connection_to_database())
     var results = papa.parse("#" + data, { header: true, comments: "#" });
     await (results.data.forEach(element => {
+      if (element["Name"]!=null)
```
Comment (DRSisco) [inline backend-processing.ts]: You should put spaces around operators. 
```js
if (term != null) {
```
https://standardjs.com/rules-en.html
>**Infix operators** must be spaced.

File: backend-processing.ts [hunk 2]
```diff
@@ -91,19 +92,23 @@ export default class BackendProcessing {
   }
   //Papaparse gives (maps?) with fields: Name,Position, Employment Term, etc.
   createCallForCSV(entry):string {
-    let name : String = entry.Name;
+    let name : String = entry["Name"];
     if (name.length<1){
```
Comment (DRSisco) [inline backend-processing.ts]: ditto

File: backend-processing.ts [hunk 3]
```diff
@@ -91,19 +92,23 @@ export default class BackendProcessing {
...
-    let position = entry.Position2;
-    let employer = entry["Current Employer"];
+    let firstPosition = entry["Portfolio Company Position"];
+    let position = entry["New Position"]
+    let employer = entry["New Employer"];
     let term = entry["Employment Term"]
+    term = this.convertDates(term)
+    let sterm = ""
+    let eterm = ""
+    if (term!=null){
```
Comment (DRSisco) [inline backend-processing.ts]: ditto

File: backend-processing.ts (hunk 1 above)
Comment (DRSisco) [reply backend-processing.ts]: Note that this could be undefined i.e, Name wasn't an included header

Comment (pastorsj) [reply backend-processing.ts]: Sounds like a tool like prettier could help here...

**Pull request merged**